DependencyInjection/CmfSeoExtension.php

```diff
@@ -251,6 +251,12 @@ private function loadErrorHandling($config, ContainerBuilder $container)
                 $container->removeDefinition('cmf_seo.error.suggestion_provider.'.$group);
             }
         }
+
+        $configurations = array();
+        $configurations['templates'] = isset($config['templates']) ? $config['templates'] : array();
+        $configurations['excluded_paths'] = isset($config['excluded_paths']) ? $config['excluded_paths'] : array();
```

Comment:
i would use a RequestMatcher and therefore have `excludes` which is a whole hashmap of filter criteria